src/test/testing_assert.cpp

```diff
@@ -118,42 +118,52 @@ ::testing::AssertionResult check_table_equal(const Table& tleft, const Table& tr
     std::sort(right.begin(), right.end());
   }
 
+  bool has_failed = false;
   for (unsigned row = 0; row < left.size(); row++)
     for (ColumnID col{0}; col < left[row].size(); col++) {
       if (is_null(left[row][col]) || is_null(right[row][col])) {
         EXPECT_TRUE(is_null(left[row][col]) && is_null(right[row][col]));
+        has_failed = true;
```

Comment:
I think this causes it to fail EVERY time. We'd have to basically do our 'own' `EXPECT_TRUE`, `EXPECT_EQ`, etc., which sets the has failed. In this case here for example:

`if (! ( is_null(left[row][col]) && is_null(right[row][col]) ) ) { has_failed = true; }`